frontend/src/metabase/query_builder/components/NativeQueryEditor.jsx

```diff
@@ -336,6 +336,12 @@ export default class NativeQueryEditor extends Component {
     }
   }
 
+  _retriggerAutocomplete = _.debounce(() => {
+    if (this._editor.completer?.popup?.isOpen) {
+      this._editor.execCommand("startAutocomplete");
+    }
+  }, 700);
```

Comment:
Yes, this is still hardcoded. But we can iterate on the right delay once this approach is considered worthy to refine further.